### PR TITLE
dnsdist: create user from the RPM package to drop privs

### PR DESCRIPTION
**Pull request by pieterlexis**
And add a paragraph about this in the readme.

### CODE REVIEW

File: build-scripts/build-dnsdist-rpm [hunk 1]
```diff
@@ -115,6 +114,41 @@ install -d %{buildroot}/%{_sysconfdir}/dnsdist
...
+%pre
+getent group dnsdist >/dev/null || groupadd -r dnsdist
+getent passwd dnsdist >/dev/null || \
+	useradd -r -g dnsdist -d / -s /sbin/nologin \
+	-c "dnsdist user" dnsdist
+exit 0
+
+
+%post
+%if 0%{?el6}
```
Comment (rubenk) [inline build-scripts/build-dnsdist-rpm]: No biggie, I guess this will work too, but usually we use  ```is %if 0%{?rhel} == 6```

**Comment by rubenk** [comment]
Please add Requires(pre): shadow-utils since you're using useradd

File: build-scripts/build-dnsdist-rpm (hunk 1 above)
Comment (pieterlexis) [inline build-scripts/build-dnsdist-rpm]: I took this from the existing EPEL specfile, but can change this.

Comment (rubenk) [inline build-scripts/build-dnsdist-rpm]: Nah, it's fine.

**Pull request merged**